zwave_js_server/model/controller/statistics.py

```diff
@@ -42,19 +42,85 @@ def nlwr(self) -> Optional[RouteStatistics]:
         return self._nlwr
 
 
-class ControllerStatisticsDataType(TypedDict):
+class ChannelRSSIDataType(TypedDict):
+    """Represent a channel RSSI data dict type."""
+
+    average: int
+    current: int
+
+
+class BackgroundRSSIDataType(TypedDict, total=False):
+    """Represent a background RSSI data dict type."""
+
+    # https://github.com/zwave-js/node-zwave-js/blob/master/packages/zwave-js/src/lib/controller/ControllerStatistics.ts#L40
+    timestamp: int  # required
+    channel0: ChannelRSSIDataType  # required
```

Comment:
Let's use `Required` from `typing_extensions`

Reply:
I am planning to come back and do this all as part of https://github.com/home-assistant-libs/zwave-js-server-python/issues/590 where I will review the interfaces across zwave-js, zwave-js-server, and the lib

Reply:
Ok.